src/applications/check-in/locales/en/translation.json

```diff
@@ -303,5 +301,11 @@
   "or-you-can-still-file-your-claim": "Or you can still file your claim within 30 days through the Beneficiary Travel Self-Service System (BTSSS).",
   "claim-submitted_one": "Claim submitted",
   "claim-submitted_other": "Claims submitted",
-  "fpo-header": "Placeholder header"
+  "fpo-header": "Placeholder header",
+  "if-you-traveled-by-bus-train-taxi-or-other--help-text": "<0>If you traveled by bus, train, taxi, or other authorized public transportation, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). Or you can use VA Form 10-3542 to submit a claim by mail, fax, email, or in person.",
```

Comment:
Another thing to note, if these need new translations, then the best practice would be to create a new key so that it comes up in the diff that we generate to know which strings to send for translation.